azure-iot-device/azure/iot/device/iothub/pipeline/pipeline_ops_iothub.py

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
-from azure.iot.device.common.transport.pipeline_ops_base import PipelineOperation
+from azure.iot.device.common.pipeline.pipeline_ops_base import PipelineOperation
```

Comment:
`azure.iot.device.common.pipeline` package should also be set up to export it's classes through the `__init__.py` interface unless there's a good reason not to IMO.

Reply:
I added the base classes to __init__.py, but I left the module a-is for namespacing of events, ops, and stages (e.g. `pipeline_ops_base.Connect` instead of `Connect`)

---
In reply to: [287150587](https://github.com/Azure/azure-iot-sdk-python-preview/pull/95#discussion_r287150587) [](ancestors = 287150587)